
Debian: in BTS
Upstream: see upstream-only/xorg-macros_kbsd-gnu.diff

diff -ur -x configure -x changelog xutils-7.0.0.old/fslsfonts-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/fslsfonts-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/fslsfonts-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:44.000000000 +0100
+++ xutils-7.0.0/fslsfonts-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:24:41.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/luit-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/luit-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/luit-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:45.000000000 +0100
+++ xutils-7.0.0/luit-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:25:16.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/rstart-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/rstart-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/rstart-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:48.000000000 +0100
+++ xutils-7.0.0/rstart-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:25:16.000000000 +0200
@@ -979,40 +979,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/scripts-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/scripts-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/scripts-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:50.000000000 +0100
+++ xutils-7.0.0/scripts-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:25:17.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/sessreg-X11R7.0-1.0.0/aclocal.m4 xutils-7.0.0/sessreg-X11R7.0-1.0.0/aclocal.m4
--- xutils-7.0.0.old/sessreg-X11R7.0-1.0.0/aclocal.m4	2006-03-20 03:42:48.000000000 +0100
+++ xutils-7.0.0/sessreg-X11R7.0-1.0.0/aclocal.m4	2006-04-19 12:25:17.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/showfont-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/showfont-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/showfont-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:50.000000000 +0100
+++ xutils-7.0.0/showfont-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:25:17.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
diff -ur -x configure -x changelog xutils-7.0.0.old/xfsinfo-X11R7.0-1.0.1/aclocal.m4 xutils-7.0.0/xfsinfo-X11R7.0-1.0.1/aclocal.m4
--- xutils-7.0.0.old/xfsinfo-X11R7.0-1.0.1/aclocal.m4	2006-03-20 03:42:49.000000000 +0100
+++ xutils-7.0.0/xfsinfo-X11R7.0-1.0.1/aclocal.m4	2006-04-19 12:25:17.000000000 +0200
@@ -1136,40 +1136,40 @@
 
 if test x$APP_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	APP_MAN_SUFFIX=1x ;;
+	linux* | k*bsd*-gnu)	APP_MAN_SUFFIX=1x ;;
 	*)	APP_MAN_SUFFIX=1  ;;
     esac
 fi
 if test x$APP_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	APP_MAN_DIR='$(mandir)/man1' ;;
+	linux* | k*bsd*-gnu)	APP_MAN_DIR='$(mandir)/man1' ;;
 	*)	APP_MAN_DIR='$(mandir)/man$(APP_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$LIB_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_SUFFIX=3x ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_SUFFIX=3x ;;
 	*)	LIB_MAN_SUFFIX=3  ;;
     esac
 fi
 if test x$LIB_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	LIB_MAN_DIR='$(mandir)/man3' ;;
+	linux* | k*bsd*-gnu)	LIB_MAN_DIR='$(mandir)/man3' ;;
 	*)	LIB_MAN_DIR='$(mandir)/man$(LIB_MAN_SUFFIX)' ;;
     esac
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	linux*)		FILE_MAN_SUFFIX=5x ;;
+	linux* | k*bsd*-gnu)		FILE_MAN_SUFFIX=5x ;;
 	solaris*)	FILE_MAN_SUFFIX=4  ;;
 	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
     case $host_os in
-	linux*)	FILE_MAN_DIR='$(mandir)/man5' ;;
+	linux* | k*bsd*-gnu)	FILE_MAN_DIR='$(mandir)/man5' ;;
 	*)	FILE_MAN_DIR='$(mandir)/man$(FILE_MAN_SUFFIX)' ;;
     esac
 fi
